src/ru/mangahub/src/eu/kanade/tachiyomi/extension/ru/mangahub/Mangahub.kt

```
@@ -0,0 +1,123 @@
+        element.select("div.b-status-label__one > span.b-status-label__name.b-status-label__name-translated").size != 0
+                && element.select("div.b-status-label__one > span.b-status-label__name.b-status-label__name-updated").size == 0 -> SManga.COMPLETED
+        element.select("div.b-status-label__one > span.b-status-label__name.b-status-label__name-updated").size != 0 -> SManga.ONGOING
+        else -> SManga.UNKNOWN
+    }
+
+    override fun chapterListSelector() = "div.b-catalog-list__elem"
+
+    override fun chapterFromElement(element: Element): SChapter {
+        val urlElement = element.select("div.b-ovf-table__elem > a").first()
+        val chapter = SChapter.create()
+        chapter.name = urlElement.text()
+        chapter.date_upload = element.select("div.b-catalog-el__date-val").text()?.let {
+            SimpleDateFormat("dd.MM.yyyy", Locale.US).parse(it).time
+        } ?: 0
+        val url = element.select("div.b-ovf-table__elem a").first().attr("href")
+        chapter.setUrlWithoutDomain(url)
+        return chapter
+    }
+
+    override fun prepareNewChapter(chapter: SChapter, manga: SManga) {
+        val basic = Regex("""Глава\s([0-9]+)""")
+        when {
+            basic.containsMatchIn(chapter.name) -> {
+                basic.find(chapter.name)?.let {
+                    chapter.chapter_number = it.groups[1]?.value!!.toFloat()
+                }
+            }
+        }
+    }
+
+    override fun pageListParse(document: Document): List<Page> {
+        val pictures = document.select("div.b-reader.b-reader__full").attr("data-js-scans").replace("&quot;", "\"").replace("\\/", "/")
+        val r = Regex("""\/\/([\w\.\/])+""")
+        val pages = mutableListOf<Page>()
+        var index = 0
+        r.findAll(pictures).forEach {
+            pages.add(Page(index=index, imageUrl="http:${it.value}"))
+            index++
+        }
+
+        return pages
+    }
+
+    override fun imageUrlParse(document: Document) = ""
```

Is this used? If not, maybe throw an exception?